tests/test_integration.py

```
@@ -49,6 +49,91 @@ def test_train_and_run(
+                tiny_config,
+                "--output_dir",
+                str(tmp_path),
+                "--output_root",
+                "train_resuming",
+                "--model",
+                "https://github.com/Noble-Lab/casanovo",
+                "--load_all_states",
+            ]
+
+            result = run(train_args)
+            assert result.exit_code != 0
```

Ruff (0.15.2)

[warning] `pytest.warns()` block should contain a single simple statement

(PT031)